### PR TITLE
Brian8ho/plu 39 install and implement cloudinary sdk

### PR DESCRIPTION
**Pull request by 8rianHo**
Description:
- added cloundinary.js file to upload profile photos
- updated post user route to allow photo upload
- updated postman collection to use form data
- added profilePicture to the user schema in mongoose/mongo

note: one of us has a linting issue @ you

This merge to main will _definitely_ break the UI...but v2 incoming so yolo

### CODE REVIEW

File: api/routes/users.js [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
 
-const admin = require('./admin');
+const admin = require("./admin");
 const Users = require("../../models/users");
+const upload = require("../utils/cloudinary");
 
 const router = express.Router({ mergeParams: true });
 
-router.use('/admin', admin);
+const uploadMiddleware = multer({
```
Comment (lujasmine) [inline api/routes/users.js:11]: Could put this in a separate file? Not a blocker 👌

File: api/routes/users.js [hunk 2]
```diff
@@ -1,15 +1,29 @@
-const log = require('loglevel');
-const express = require('express');
+const log = require("loglevel");
+const multer = require("multer");
+const express = require("express");
 
-const admin = require('./admin');
+const admin = require("./admin");
 const Users = require("../../models/users");
+const upload = require("../utils/cloudinary");
```
Comment (lujasmine) [inline api/routes/users.js:7]: Could name to `cloudinaryUpload`?

**Review by lujasmine — APPROVED** [review]
Thanks for linting 🌟

File: api/routes/users.js (hunk 1 above)
Comment (8rianHo) [reply api/routes/users.js]: please see https://github.com/8rianHo/vietnamese-directory-server/pull/17/commits/686a0945f896c3868df8e02c88509213a7e6f3e4

File: api/routes/users.js (hunk 2 above)
Comment (8rianHo) [reply api/routes/users.js]: please see https://github.com/8rianHo/vietnamese-directory-server/pull/17/commits/686a0945f896c3868df8e02c88509213a7e6f3e4

**Review by lujasmine — APPROVED** [review]
💯

**Pull request merged**